atd-vze/src/views/Crashes/Recommendations.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from "react";
+import { useQuery, useMutation } from "@apollo/react-hooks";
+import { Card, CardHeader, CardBody, CardFooter, Table, Button } from "reactstrap";
+import { recommendationsDataMap } from "./recommendationsDataMap";
+import { GET_RECOMMENDATIONS } from "../../queries/recommendations";
+import { useAuth0, isReadOnly } from "../../auth/authContext";
+
+// declare fatality review board recommendations component
+const Recommendations = ({ crashId }) => {
+
+  // fetch data from database using graphQL query
+  const { loading, error, data, refetch } = useQuery(GET_RECOMMENDATIONS, {
+    variables: { crashId },
+  });
+
+  if (loading) return "Loading...";
+  if (error) return `Error! ${error.message}`;
+
+  // if crash has data in recommendations table, display it
+  const displayData = (field) => {
+    if (data.recommendations[0]) {
+        if (field === "partner") {
+            return data.recommendations[0].atd__coordination_partners_lkp.description
+        } else if (field === "status") {
+            return data.recommendations[0].atd__recommendation_status_lkp.description
+        } else if (field === "note") {
+            return data.recommendations[0].text
+        }}
+  }
+
+  const fieldConfig = recommendationsDataMap[0];
+
+    return(
+        <Card>
+            <CardHeader>{fieldConfig.title}</CardHeader>
+            <CardBody>
+                <Table>
```

Comment:
do you need to handle when there this is no recommendation? If so, conditionally returning a message inside the card body prob makes the most sense, along with only conditionally rendering the `Table` component if there is a recommendation.

Reply:
next up i will be adding editing capabilities to the component, so the table component will need to be rendered even if there is no recommendation (https://github.com/cityofaustin/atd-data-tech/issues/9519). so i think it makes sense to just leave it as is for now?

Reply:
ah, I see. humble opinion, but the right approach is probably to move your labels into a proper header proper header row (reactstrap may have a component for it)
```html
 <thead><tr>Coordination Partner<th>Status</th><th>Note</th></tr></thead>
```
and in cases where there is no recommendation, render the "add recommendation" form in a row below it.